flow/environment.py

```diff
@@ -191,6 +198,73 @@ def get_config_value(cls, key, default=flow_config._GET_CONFIG_VALUE_NONE):
         """
         return flow_config.require_config_value(key, ns=cls.__name__, default=default)
 
+    @classmethod
+    def _get_omp_prefix(cls, operation):
+        """function for getting the omp prefix based on proper directives.
```

Comment:
```suggestion
        """Get the OpenMP prefix based on the `omp_num_threads` directive.
```
While *omp* is a legitimate abbreviation for OpenMP, I think we should be a bit more precise on the documentation.

Furthermore, the first line of the doc-string should be a proper sentence and therefore also be capitalized.